platform-core-module/src/main/java/com/softicar/platform/core/module/program/execution/AGProgramExecutionTable.java

```diff
@@ -34,11 +34,11 @@ public void customizeAttributeProperties(IEmfAttributeList<AGProgramExecution> a
 
 		attributes//
 			.editAttribute(AGProgramExecution.STARTED_AT)
-			.setDisplayFactory(value -> new EmfDayTimeDisplay(value, DayTime::toString));
+			.setDisplayFactory(value -> new EmfDayTimeDisplay(value, DayTime::toDisplay));
```

Comment:
- I am afraid that may developers will forget to call `toDisplay` instead of `toString`.
- are we sure `toString` should not respect `CurrentLocale`?

Reply:
In the long run, I think `toString` should indeed respect `CurrentLocale`, and provide output that is formatted accordingly - I see no way around this. Yet, I considered that too dangerous for now:
Before changing the output of `toString`, we must somehow ensure that we currently _never_ use `toString` when we in fact need an ISO-formatted date (that's precisely why it's a bad practice to rely on the output of `toString` in a non-user-facing context, btw).
As long as we don't find an efficient way to detect this, we're kind of stuck in the current middle ground, in which locale-aware date formatting is _possible_ (via `toDisplay`) while unfortunately not being the default behavior.

Reply:
- if we don't change the default behavior of `toString` now, we will never change it ;) but see next point
- I agree that changing `toString` is dangerous, but then we should at least ensure that anything related to `IDisplayString` will always use `toDisplay` and not `toString`, again, see `I18nFormatArgumentsLocalizer` and consider adding an `if (argument instanceof IDisplayable)` case

Reply:
Did the `I18nFormatArgumentsLocalizer` thing. We chould change the behavior of `toString` - but I think we should do so in a separate PR, so that it's easily reversible (without taking away the feature).